modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,108 @@
+= Migrate to the Version 2 of the Mocking Service
```

Comment:
Should we strike the "the" here to be consistent with the subtitle below?

> How to Migrate an API-Specification Project to Version 2 of the Mocking Service

Reply:
Thanks.